.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+**What type of PR is this?**
+> Uncomment only one:
+>api-change
+>bug
+>cleanup
+>design
+>documentation
+>failing-test
+>feature
+
+**What this PR does / why we need it**:
+
+**Special notes for your reviewer**:
+
+**Does this PR introduce a user-facing change?**:
```

Comment:
This is too vague. Almost everything is a user-facing change. Does this mean API/CRD/cli changes only? Is the submitter supposed to provide justification on why it is changing?

Reply:
Refactored a bit. Removed the `Special notes for your reviewer` section since same text can be given in the "What this PR does.." section.